vhdbuilder/packer/list-images.ps1

```diff
@@ -96,7 +99,17 @@ foreach($image in $imageList) {
 Stop-Job  -Name containerd
 Remove-Job -Name containerd
 
-$bomList | ConvertTo-Json | set-content $imageBomJsonFilePath
+$imageBom=$(echo $bomList | ConvertTo-Json)
+
+$listResult = @"
+{
+        "sku": "$windowsSKU",
+        "imageVersion": "$windowsImageVersion",
+        "imageBom": $imageBom
+}
+"@
+
+echo $listResult | ConvertFrom-Json | ConvertTo-Json -Depth 3 | set-content $imageBomJsonFilePath
```

Comment:
Why do we need `ConvertTo-Json -Depth 3`? Please check the generated bom json and we may need to remove the extra blanks.

Reply:
Default depth of ConvertTo-Json is 2. Since we add one more layer, we need to set the depth to 3 to keep the content of imageBom the same as before.